plugins/polio/js/src/domains/VaccineModule/StockManagement/Table/useVaccineStockManagementTableColumns.tsx

```diff
@@ -55,7 +60,9 @@ export const useVaccineStockManagementTableColumns = (): Column[] => {
                         value={settings.row.original.stock_of_unusable_vials}
```

Comment:
In addition to hiding the column when it is only nOPV vaccine why not put '---' placeholder on bOPV vaccine in the case in the table results there are also other vaccine like nOPV 
![Screenshot-from-2024-10-04-14-58-12](https://github.com/user-attachments/assets/dbe89834-2595-4ef6-a1bb-14b7f0ce386d)